requirements/base.txt

```diff
@@ -21,9 +21,9 @@ aiohttp==3.6.0 \
     --hash=sha256:e0fe698d1e6a852a27a88d2844a1a63839ee764d7cf214fd58cbea480407cc1d \
     --hash=sha256:fa155e309cc2277d6f9d099aecaf3ce78d86a31f5a62a994debc872e4c34ddf4 \
     # via scriptworker, taskcluster
-arrow==0.14.7 \
-    --hash=sha256:4bfacea734ead51495dc47df00421ecfd4ca1f2c0fbe58b9a26eaeddedc31caf \
-    --hash=sha256:67f8be7c0cf420424bc62d8d7dc40b44e4bb2f7b515f9cc2954fb36e35797656
+arrow==0.15.0 \
```

Comment:
Arrow 0.15.0 (may be broken](https://github.com/mozilla-releng/scriptworker/issues/394). We can likely leave it and fix in scriptworker.

Reply:
ok, I can revert this.

Reply:
It looks like we're not affected by it, so feel free to leave it.